File: labSocketsFestivalCSCompleted/labSocketsFestivalCSV2/ConsoleApplication1/model/Cashier.cs
using System;

namespace model
{
    [Serializable]
    public class Cashier:Entity<int>
    {
        private string username;

        private string password;
        
        
        public Cashier(int id) : base(id)
        {
            
        }

        public Cashier(int id,string username, string password) : base(id)
        {
            this.username = username;
            this.password = password;
        }
        
        public string Username
        {
            get { return username; }
            set { username = value; }
        }

        public string Password
        {
            get { return password; }
            set { password = value; }
        }
    }
}

File: labRepositoryJava/src/main/java/org/example/model/Cashier.java
package org.example.model;

public class Cashier extends Entity<Integer>{

    private String username;
    private String password;

    public Cashier(String username,String password,Integer integer) {
        super(integer);
        this.username=username;
        this.password=password;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
}
